app/src/components/SearchBar.jsx

```diff
@@ -2,17 +2,23 @@ import React, { useState } from "react";
 import "../stylesheets/searchBar.scss";
 import closeSvg from '../assets/close.svg'
 import SearchSvg from '../assets/search.svg'
+import useFetch from "../hooks/useFetch";
+import { Link } from 'react-router-dom';
 
 
-function SearchBar({ placeholder, data }) {
+
+function SearchBar({ placeholder }) {
     const [filteredData, setFilteredData] = useState([]);
```

Comment:
what is the filtered data? is it artists? events? both?

Reply:
Filtered data är de filtrerade sökresultaten (matchningarna mellan wordEntered och den faktiska datan som finns i databasen). I nuläget har vi bara lagt koppling till artister i databasen, men det ska även gå att söka på venues/events

Läs rad 54.